client/schemes/jellyfinScheme.ts

```diff
@@ -20,33 +21,46 @@ export default class JellyfinScheme {
     this.$auth = auth;
 
     this.options = Object.assign({}, {}, options);
+    this.mount();
   }
 
   _setToken(token: string): void {
     // Set Authorization token for all axios requests
     this.$auth.ctx.app.$axios.setHeader('X-Emby-Authorization', token);
   }
 
+  _setBaseUrl(serverUrl: string): void {
+    this.$auth.ctx.app.$axios.setBaseURL(serverUrl);
+  }
+
   _clearToken(): void {
     // Clear Authorization token for all axios requests
     this.$auth.ctx.app.$axios.setHeader('X-Emby-Authorization', false);
   }
 
   _setRememberMe(value: boolean): void {
     // Sets the remember me key which is used to relogin someone
-    this.$auth.$storage.setUniversal('rememberMe', value);
+    this.$auth.$storage.setState('rememberMe', value);
```

Comment:
As explained above, imo we should have everything handled by Vuex only to keep consistency and handle the persistence using ``vuex-persistedstate``, which allows us to have complete control of how data is stored and consistency if we ever need to access or subscribe to mutations in the accessToken or servers, for instance.

Also, saving to Vuex before was useless as we didn't set the ``auth`` namespace as one of the persisted ones.

Reply:
I disagree, we shouldn't circumvent how Nuxt Auth works just because we persist some other stuff.

Then, we may as well completely remove Nuxt Auth and do everything ourselves, if we follow your logic...

Nuxt Auth still uses the store, but it manages it itself, which is what it's expected to do. Again, this adds complexity to the implementation for no reason, and makes it harder for someone coming in to understand what the hell is happening and why, while letting Nuxt Auth do its thing and staying as close as possible to the upstream schemes makes it pretty easy to simply go there and look at docs.